assets/app/features/upgrade/templates/landing-page_step3.partial.jade

```diff
@@ -0,0 +1,12 @@
+.upgrade(ng-class="{'active': upgradeLandingVm.mode.valid}")
+    .step-title.clearfix
+        crowbar-circle.pull-left(ng-class="{'active': upgradeLandingVm.mode.valid}") 3
```

Comment:
With this approach (i.e. binding css classes to models directly) you don't need a directive at all. To make better use of the possibilities you get with a directive, you should make `active` an attribute of the directive, bind model to this and use link function to attach behavior to changes in `active` model (i.e. update css classes).